test/toolshed/unix_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Toolshed.UnixTest do
+  use ExUnit.Case
+  alias Toolshed.Unix
+
+  test "date/0 returns current date in unix format" do
+    assert Unix.date() == unix_date_output()
+  end
+
+  defp unix_date_output do
+    "date"
+    |> System.cmd(["-u"], env: %{"LC_ALL" => "C"})
+    |> elem(0)
+    |> String.trim()
```

Comment:
I avoid using `elem/2`in pipes since it almost always lets an error tuple propagate which should have been caught earlier. In this case, error exit codes from `System.cmd` are ignored. It's not much code, so I don't think it would be a big deal here, but in larger code bases, it gets harder to track down.

Here's how I'd write it:

```suggestion
    {result, 0} = System.cmd("date", ["-u"], env: %{"LC_ALL" => "C"})
    String.trim(result)
```